notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>please use this form for the service account:</p><p> </p><p>if (</p><p>&nbsp;&nbsp;SERVICE_ACCOUNT == ""</p><p>&nbsp;&nbsp;or SERVICE_ACCOUNT is None</p><p>&nbsp;&nbsp;or SERVICE_ACCOUNT == "[your-service-account]"</p><p>):</p><p>&nbsp;&nbsp;# Get your GCP project id from gcloud</p><p>&nbsp;&nbsp;shell_output = !gcloud auth list 2&gt;/dev/null</p><p>&nbsp;&nbsp;SERVICE_ACCOUNT = shell_output[2].replace("*", "").strip()</p><p>&nbsp;&nbsp;print("Service Account:", SERVICE_ACCOUNT)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>